### ISSUE TITLE
Interactive figure is too big for one column

### ISSUE THREAD

**Issue by abostroem** [AASJournals/AASTeX60#109]
The interactive figure box is spanning almost 2 columns (see attached screenshot) when using the following command to add an animation to my manuscript (using aastex 6.3). 

```
\begin{figure}
    \begin{interactive}{animation}{BPASS_3d.mp4} %animation
    \includegraphics[width=\columnwidth]{binary_star_upper_lim_3d.pdf} %preview
    \end{interactive}
    \caption{An animated version of this figure is available...}
    \label{fig:3d}
\end{figure}
```

![Screen Shot 2020-02-13 at 10 40 39 AM](https://user-images.githubusercontent.com/2522639/74467917-d23f4980-4e4e-11ea-802f-4badfa5353cf.png)

**Comment by augustfly** [comment]
Yeah, I think this is a bug in the internal function `\highlightfigure`.  The line that reads `\textwidth` should use `\columnwidth` instead. I'm not even sure that is going to work perfectly given the hacky way this is written. 

https://github.com/AASJournals/AASTeX60/blob/d59c29ea6d996a45b499ff95fb0379e8c56296eb/cls/aastex63.cls#L5810-L5815

should be changed to

```
\long\def\highlightfigure#1{\centerline{
\fboxrule=4pt
\fboxsep=12pt
\fcolorbox{ltblue}{white}{\hbox to
.93\columnwidth{\hss$\vcenter{\advance\hsize -24pt #1}$\hss}}
}}
```

**Comment by abostroem** [comment]
I can confirm that this change fixed the issue for me

**Comment by abostroem** [comment]
However, out of curiosity I tested figure* and the one columnwidth blue box is now centered in the middle of two columns overlapping with the two column figure. This probably needs a more robust workaround to make both work

**Comment by augustfly** [comment]
@abostroem exactly right! thank you for reporting this and testing that alternate (failed) use case.
